src/read_csv.py

```diff
@@ -3,13 +3,16 @@
 import os
 
 
-def readFile(file):
-    countylist = []
-    newspaperlist = []
+def read_in_counties_newspapers(file):

```

Comment:
rename to `read_county_to_newspaper_csv`

Reply:
done